apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -0,0 +1,316 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-sso-saml',
+  title: 'Single Sign-On with SAML 2.0',
+  description: 'Use Single Sign-On (SSO) authentication with SAML 2.0',
+  video: 'https://www.youtube.com/v/em1cpOAXknM',
+}
+
+Supabase Auth supports enterprise level Single Sign-On (SSO) using the SAML 2.0 protocol. In this document you will learn how you can manage connections to compatible identity providers.
+
+<Admonition type="info">
+This is an early beta release of these APIs. We are still working on adding support and a great developer experience in the CLI and Dashboard.
+
+If you are comfortable using these APIs and would like to try out SSO with SAML 2.0 for your project, please [open a support ticket](https://app.supabase.com/support/new).
+
+We do not believe these APIs will change significantly until the feature is generally available, but we do reserve the right to modify them. Projects in the beta will be notified of any changes.
+
+</Admonition>
+
+## Terminology
+
+When working with SAML and SSO, it often can feel like there's too many acronyms to keep track of. Here's what all of them mean.
+
+- **Identity Provider**, **IdP** or **IDP**  
+  This is software that manages user accounts at a company or organization. It is able to verify the identity of a user and exchange that information with your Supabase project. Commonly used identity providers are: Microsoft Active Directory (Azure AD, Microsoft Entra), Okta, Google Workspaces (GSuite), PingIdentity, OneLogin, and many others.
```

Comment:
```suggestion
  This is software that manages user accounts at a company or organization. It can verify the identity of a user and exchange that information with your Supabase project. Commonly used identity providers are: Microsoft Active Directory (Azure AD, Microsoft Entra), Okta, Google Workspaces (GSuite), PingIdentity, OneLogin, and many others.
```

/nit